TESTING_WITH_IAF-TEST.md

```diff
@@ -37,7 +37,11 @@ To make sure our database can be used, we'll have to run an ant script. Navigate
 * _/setupDB/Oracle/create_user.xml_
 * _/setupDir/setupDir.xml_
 
-## 4. Running the test scenarios
+## 4. In ibis-adapterframework-test, select Oracle database
+
+The ibis-adapterframeworkt-test project supports multiple databases. In your local checkout, please edit the code to select Oracle as the database. Please edit file "src/main/resources/DeploymentSpecifics.properties". Update the line `jdbc.datasource.default=${jdbc.datasource.h2}` to make it `jdbc.datasource.default=${jdbc.datasource.oracle}`. This change should not be checked in!
+
+## 5. Running the test scenarios
```

Comment:
Hey Martijn,

Ik denk dat handig is dit soort wijzigingen in je catalina.properties te zetten. Je moet namelijk ook (sommige) datasources configureren (denk aan endpoints username/passwords). 
De property is vooral ontstaan om met een enkele war file, met meerdere databases te kunnen testen, puur door het aanpassen van deze property op jvm niveau.